src/components/comment/CommentItem.tsx

```diff
@@ -39,14 +41,26 @@ const Button = styled.div`
 
 function CommentItem({ comment }: any) {
   const dispatch = useDispatch();
+  const isButtonDisabled = useSelector(
+    (state: RootState) => state.comment.buttonDisabled,
+  );
+  const updateHandler = () => {
+    dispatch(editComment(comment.id));
+    disableButton(dispatch);
+  };
+
   return (
     <Comment key={comment.id}>
```

Comment:
이 부분을 제거하고 Comments.tsx 에서 map 돌릴 때 comment.id로 key를 지정해주면 좋지 않을까 싶네요 !
```suggestion
    <Comment>
```